lms/djangoapps/entitlements/models.py

```diff
@@ -0,0 +1,69 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from jsonfield import JSONField
+
+from .registry import registry
+from .scope import ScopeFactory
+
+
+class EntitlementModel(models.Model):
```

Comment:
@mattdrayer that's a naming question, but my motivation behind renaming it to `EntitlementModel` (initially it was just `Entitlement`) is that it is not actually an Entitlement - it is more of a recipe stored in the DB to create actual `Entitlement`.

To make things clear:
* Entitlement is an object that can check its scope.
* EntitlementModel is how that object is saved to the DB.

I thought explicitly mentioning that model is not an Entitlement would be helpful to get the overall schema.